modus/overview.mdx

```diff
@@ -8,50 +8,72 @@ sidebarTitle: "Overview"
-integrate models in different forms, whether generative large language models or
-classical machine/deep learning models.
+## Key features
 
-Your app might be a simple create, read, update, and delete (CRUD) function that
-has a single model to identify similar entries. Or, it could be a complex
-agentic reasoning system that chains dozens of models and data sources together.
-**Modus creates a way of working with models that scales with your needs.**
+Modus is built from the ground up for agent development.
 
-Modus exists to make it easier for you to build the apps of your dreams.
+| Feature                     | Description                                                                                      |
+| --------------------------- | ------------------------------------------------------------------------------------------------ |
+| WebAssembly powered runtime | A small, portable, and highly efficient execution engine for deploying agents anywhere           |
+| Flexible tool support       | Consume Model Context Protocol (MCP) servers or build your own custom tools to power your agents |
+| Triggered agent actions     | Define and trigger specific actions for your agents to perform                                   |
+| Built-in agent memory       | Support for both short-term and long-term agent memory, powered by knowledge graphs              |
+| Agent operations            | Built-in inference logging and traces for agent-level observability                              |
+| Serverless scale-to-zero    | Each agent spins up on demand and scales to zero in idle state                                   |
+| Fully open source           | No vendor lock-in, but designed for low cold-start times and rapid scaling                       |
 
 <Note>
   Modus is a multi-language framework. It currently includes support for Go and
   AssemblyScript, a WebAssembly compatible TypeScript-like language. Additional
   language support is in development.
 </Note>
 
-## What is Modus good for? {/* vale Google.Contractions = NO */}
+## When to use Modus
 
-We designed Modus primarily as a general-purpose app framework, it just happens
-to treat models as a first-class component. With Modus you can use models, as
-appropriate, without additional complexity.
+### Rapid prototyping to scaling with confidence
 
-However, Modus is best for apps that require sub-second response times. We've
-made trade-offs to optimize for speed and simplicity.
+Start with one-line agent definitions using **AgentKit** - no infrastructure
+setup, no complex wiring. You get streaming UI, context management, and logging
+out of the box. Once your agent logic is solid, deploy via the same framework.
+Because agents are WebAssembly based and serverless, they incur zero cost when
+idle and start fast under load.
 
-For more inspiration, check out the
-[Modus recipes](https://github.com/hypermodeinc/modus-recipes).
+### Language-agnostic logic, safely
```

Comment:
I wonder if it's worth adding a small demo code block, containing a 'Hello World' function, with Golang and AssemblyScript tabs, so readers can see that we support both runtimes and the different syntax in each implementation.